src/components/ProgressBar/src/ProgressBar.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+	<div
+		:class="[
+			$s.ProgressBarContainer,
+			$s[`size_${size}`],
+			$s[`shape_${shape}`],
+		]"
+	>
+		<div
+			:class="[
+				$s.ProgressBar,
+				$s[`shape_${shape}`],
+			]"
+			:style="{
+				backgroundColor: color,
+				transform: `translateX(-${progressTranslate}%)`
+			}"
```

Comment:
We should move declarations to the CSS and the calculated values to be computed css properties.

`:style="computedStyles"` then a computed method that returns two properties:
```
{
  '--bar-color': this.color,
  '--fill-percent: MAX_PROGRESS - this.progress
}
```

Then in the styles declare the background property and transform you're doing inline and leverage the inline properties instead.

This is a lot cleaner way of keeping the template tight, the computed parts in JS, and the CSS declarations in styles so everything is where it's expected.